default/specials.txt

```diff
@@ -1969,6 +1681,43 @@ Special
 //##########################################################
 
 
+// parameter 1 is the upper case monster name
+// parameter 2 is the lower case monster name
```

Comment:
Is the lower-case name used for anything other than the icon file name? If not, then probably would be better labelled as such.